default/python/AI/ShipDesignAI.py

```diff
@@ -372,17 +370,15 @@ def _check_cache_for_consistency(self):
             try:
                 cached_name = fo.getShipDesign(self.design_id_by_name[designname]).name
                 if cached_name != designname:
-                    print "  WARNING: ShipID cache corrupted."
-                    print "    Expected: %s, got: %s. Repairing cache." % (designname, cached_name)
+                    warn("ShipID cache corrupted. Expected: %s, got: %s." % (designname, cached_name))
```

Comment:
This should be an error.  The cache is broken, which should never happen, even with bad data.

Reply:
This indicates a mismatch between AI cache and server ids. This is as likely an error of the server as it is of the AI code. In my opinion, this should stay on ```warn``` level.

Reply:
I'd still make it an error, because it is something that a dev needs to fix.  Although maybe not the dev who wrote the python code.